submissions/peterIyiola/css/contact.css

```diff
@@ -0,0 +1,119 @@
+.flex-container   {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    background-color: greenyellow;
+    padding: 5px;
+    justify-content: space-around;
+}
+.head {
+    background-color: greenyellow;
+    flex: row wrap;
+    flex-direction: row;
+    justify-content: space-around;
+}
+.pp {
+    padding-top: 13px;
+}
+.CU {
+    padding-top: 13px;
+}
+.SU {
+    padding-top: 13px;
+}
+
+.firstname  {
+    border-radius: 25px;
+}
+a   {
+    text-decoration: none;
+}
+.CD {
+    display: block;
+    float: left;
+    margin-left: 200px;
+    margin-top: 100px;
+    line-height: 1.5px;
+    text-align: center;
+}
+h4  {
+    color: grey;
+}
+.fa {
+    padding: 10px;
+    font-size: 30px;
+    width: 15px;
+    text-align: center;
+    margin: 5px;
+    border-radius: 5px;
+  }
+  .fa:hover {
+    opacity: 0.7;
+  }
+  .fa-facebook {
+    background: #3B5998;
+    color: white;
+  }
+  .fa-twitter {
+    background: #55ACEE;
+    color: white;
+  }
+  .fa-linkedin {
+    background: #007bb5;
+    color: white;
+  }
+  .fa-youtube {
+    background: #bb0000;
+    color: white;
+  }
+.field  {
+    margin-left: 700px;
+    margin-top: 50px;
+    width: 150px;
+    margin-bottom: 50px;
+}
+.message    {
+    margin-left: 150px;
+}
+.HL {
+    text-align: center;
+}
+.SM {
+    margin-left: 290px;
+    margin-top:5px;
+    color: goldenrod;
+     cursor: pointer;
+}
+.SM:hover  {
```

Comment:
let your class names be descriptive what it is doing or where it is been used. @Iyiola-stack Rename the class in html as well